src/platforms/android/profiling/index.mdx

```diff
@@ -50,3 +52,17 @@ In `AndroidManifest.xml`:
 The `io.sentry.traces.profiling.sample-rate` setting is _relative_ to the `io.sentry.traces.sample-rate` setting.
 
 </Note>
+
+## App Start Profiling
+
+When app start profiling is enabled, the whole app start process is profiled.
+This includes all methods from any `ContentProvider`, the `Application` class and the first Activity, until the <PlatformLink to="/performance/instrumentation/automatic-instrumentation/">first automatic Activity transaction</PlatformLink> is finished.
+App Start Profiling can be enabled with the manifest option `io.sentry.traces.profiling.enable-app-start` as shown above, and it will respect the `io.sentry.traces.sample-rate` and the `io.sentry.traces.profiling.sample-rate`.
+If you prefer to use a <PlatformLink to="/configuration/sampling/#setting-a-sampling-function">sampling function</PlatformLink>, a new field was added to the `TransactionContext` to specify it will be used for the app start profiling: `isForNextAppStart`.
```

Comment:
```suggestion
When app start profiling is enabled, the whole app start process is profiled.
This includes all methods from any `ContentProvider`, the `Application` class, and the first Activity, until the <PlatformLink to="/performance/instrumentation/automatic-instrumentation/">first automatic Activity transaction</PlatformLink> is finished.
App start profiling can be enabled with the manifest option `io.sentry.traces.profiling.enable-app-start` as shown above, and it will respect the `io.sentry.traces.sample-rate` and the `io.sentry.traces.profiling.sample-rate`.
If you prefer to use a <PlatformLink to="/configuration/sampling/#setting-a-sampling-function">sampling function</PlatformLink>, you can set the `isForNextAppStart` field on the `TransactionContext` to specify it will be used for app start profiling.
```

Reply:
Thanks for all of your suggestions!
Just one thing, as I guess it was not clear before. It's the SDK that sets the `isForNextAppStart` flag, and the user can check its value to make his decision.
May you have another look to check if it's more clear now?

Reply:
Oops, sorry I missed this comment. Yes, I think this is much more clear. Thanks for noticing that and making the change!